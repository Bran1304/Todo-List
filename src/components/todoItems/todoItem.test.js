import React from 'react';
import renderer from 'react-test-renderer';
import TodoItem from './todoItem';

it('renders correctly', () => {
  const tree = renderer
    .create(<TodoItem
      title="Test"
      key={1}
      id={1}
      removeItem={jest.fn()}
      onTitleChange={jest.fn()}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
