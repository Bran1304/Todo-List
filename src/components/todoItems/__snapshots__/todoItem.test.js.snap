// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="item clearfix"
>
  <div
    className="title"
  >
    <input
      defaultValue="Test"
      onChange={[Function]}
    />
  </div>
  <button
    className="removeBtn"
    onClick={[Function]}
    type="button"
  />
</div>
`;
